src/main/java/lotto/domain/NumberGenerator.java

```diff
@@ -13,23 +13,18 @@
 
     private static final int DEFAULT_START_NUMBER = 1;
     private static final int DEFAULT_END_NUMBER = 45;
+    private static final List<Integer> BASE;
 
-    private final List<Integer> base;
-
-    public NumberGenerator() {
-        this(DEFAULT_START_NUMBER, DEFAULT_END_NUMBER);
-    }
-
-    public NumberGenerator(int start, int end) {
-        base = IntStream
-                .range(start, end + 1)
+    static {
+        BASE = IntStream
+                .range(DEFAULT_START_NUMBER, DEFAULT_END_NUMBER + 1)
```

Comment:
```suggestion
                .rangeClosed(DEFAULT_START_NUMBER, DEFAULT_END_NUMBER)
```
위와 같이 표현하는 것이 더 직관적이지 않을까요